packages/esm-tb-app/src/dashboard.meta.tsx

```diff
@@ -48,9 +48,15 @@ export const tbClinicalViewDashboardMeta = {
 export const tbCasesDashboardMeta = {
   name: 'tb-cases',
   slot: 'tb-cases-dashboard-slot',
-  config: { columns: 1, type: 'grid', programme: 'tb', dashboardTitle: 'TB Treatment', icon: PillsAdd },
+  config: { columns: 1, type: 'grid', programme: 'tb', dashboardTitle: 'TB Treatment' },
   title: 'TB Treatment',
-  dashboardIcon: PillsAdd,
+};
+
+export const tptCasesDashboardMeta = {
+  name: 'tb-prevention-cases',
+  slot: 'tpt-cases-dashboard-slot',
+  config: { columns: 1, type: 'grid', programme: 'tpt', dashboardTitle: 'TB Prevention'},
+  title: 'TB Prevention',
```

Comment:
Let's align the naming here. If this is `TB` then let the const naming and slots reflect that. Replace `tpt` with `tb`

The constant naming should also reflect `tbPreventionDashboardMeta`

Reply:
@lucyjemutai So I think you should maintain the tpt naming as we spoke about yesterday.
@pirupius The title according to the wireframe is TB Prevention, having it tpt will make it easier to tell apart from TB work itself